manifests/check.pp

```diff
@@ -236,10 +238,53 @@
     publish             => $publish,
     dependencies        => $dependencies,
     custom              => $custom,
```

Comment:
I think this would create a key named `custom` with the data in it and so it should be removed. We can join the hashes like

```puppet
$content_hash = $resource_hash + $custom
```